flexmeasures/ui/templates/crud/account.html

```diff
@@ -379,21 +374,48 @@ <h3>Assets</h3>
 <script type="text/javascript">
   const basePath = window.location.origin;
   const form = document.getElementById("editaccount");
-  const toastElement = document.getElementById("toastcontainer");
+  const toastContainer = document.getElementById("toastcontainer");
   const toastDescription = document.getElementById("toastbody");
 
   function showToast(message, type) {
+    let colorClass;
+    let title;
+    if (type == "error") {
+      colorClass = "bg-danger";
+      title = "Error";
+    } else if (type == "success") {
+      colorClass = "bg-success";
+      title = "Success";
+    } else {
+      colorClass = "{{ account.primary_color }}";
```

Comment:
Did you test if this works? I believe there are no info messages I can test easily in this PR.

Because the hex code is not suitable for being used as a CSS class, as far as I can tell.

Reply:
I haven't tested this part yet since I don't have an example case, but I see your point, and I think that part won't work, I need to put it in a style property 😅